test/helpers/Constants.scala

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helpers
+
+import models.ChargeReference
+import models.declarations.{Declaration, State}
+import play.api.libs.json.{JsObject, Json}
+
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+import scala.util.Random.nextInt
+
+trait Constants {
+  val chargeReferenceNumber: Int               = 1234567890
+  val chargeReference: ChargeReference         = ChargeReference(chargeReferenceNumber)
+  def randomChargeReference(): ChargeReference = ChargeReference(nextInt(chargeReferenceNumber))
+  val correlationId: String                    = "fe28db96-d9db-4220-9e12-f2d267267c29"
+  val lastUpdated: LocalDateTime               = LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS)
+
+  val emailAddress = "testEmail@digital.hmrc.gov.uk"
+
+  val userInformation: JsObject = Json.obj(
+    "firstName"            -> "Harry",
+    "lastName"             -> "Potter",
```

Comment:
See - https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open